docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,214 +1,169 @@
+
+If you are running Windows in an AD environment, consider integrating Teleport
+with Active Directory for automatic discovery by reading [Desktop Access with
+Active Directory](./active-directory.mdx). You can also mix Active Directory
+integration with the static host definitions described below.
+
 <Details
   title="Version warning"
   opened={true}
   scope={["oss", "enterprise"]}
   scopeOnly={true}
-  min="10.2.6"
+  min="12.0"
 >
-
-The features documented on this page are available in Teleport `10.2.6` and higher.
-
+Windows Access without Active Directory is available starting from Teleport `v12`.
+Previous versions of Teleport can implement Windows Access by integrating with
+an [Active Directory domain](./active-directory.mdx).
 </Details>
 
-This guide will help you configure Teleport and Active Directory in order to
-provide secure, passwordless access to Windows desktops.
-
-<Details title="Compare Desktop Access to other RDP clients">
-
-Teleport Desktop Access is designed to be a secure access solution for Windows
-environments. Teleport implements a minimal feature set of the RDP protocol with
-security as a priority, and may not be as performant as standard RDP clients.
-Consider Desktop Access to manage access to your most sensitive Windows
-environments, not as a drop-in replacement for other tools to provide general
-access to Windows desktops.
-
-</Details>
+<Admonition type="warning">
+Teleport Desktop Access is not yet compatible with Windows Server 2022.
+</Admonition>
 
 ## Prerequisites
 
-<ScopedBlock scope={["oss", "enterprise"]}>
-- One or more hosts to run the Teleport Auth and Proxy services on.
-</ScopedBlock>
-- A server or virtual machine running a Windows Server operating system.
-  In this guide, we'll install Active Directory on this server in order
-  to support passwordless logins with Teleport to the Windows desktops
-  in the Active Directory domain.
-- A Linux host where you will run the Teleport Desktop Service.
-  <ScopedBlock scope={["oss", "enterprise"]}> This guide assumes that you will
-  run Teleport's Windows Desktop Service on a dedicated host. To install Desktop
-  Access into an existing Teleport instance running other services, see the
-  [Manual Setup](manual-setup.mdx) guide for Desktop Access.</ScopedBlock>
-- An Active Directory domain, configured for LDAPS (Teleport requires an
-  encrypted LDAP connection). Typically this means installing
-  [AD CS](https://learn.microsoft.com/en-us/windows-server/identity/ad-cs/)
+- (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
-## Step 1/2. Install Teleport
+- A Linux server to run the Teleport Desktop Access service on.
+  You can reuse an existing server running any other Teleport instance.
 
-### Set up the Teleport Auth and Proxy Services
+- (!docs/pages/includes/tctl.mdx!)
 
-(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+## Step 1/4. Prepare Windows
```

Comment:
I think this section would benefit from a short paragraph that explains the architecture of the setup we're working through in the guide. That way, it would be easier for me to form a mental model that I can use to make sense of all the instructions.

Reply:
Added below.